Makefile

```diff
@@ -20,6 +32,12 @@ code/check:
 code/fix:
 	@gofmt -w `find . -type f -name '*.go' -not -path "./vendor/*"`
 
+.PHONY: setup/prow
+setup/prow:
+	@echo Installing Operator SDK
+	@curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v$(OPERATOR_SDK_VERSION)/operator-sdk-v$(OPERATOR_SDK_VERSION)-x86_64-$(OPERATOR_SDK_OS) && chmod +x operator-sdk
+
+
```

Comment:
Should probably follow the same pattern as the integeratly operator and create a Dockerfile containing all the build tools required https://github.com/integr8ly/integreatly-operator/tree/master/openshift-ci